lib/location/location_service_utils.c

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdio.h>
+#include <zephyr/kernel.h>
+#include <assert.h>
+
+#include <zephyr/logging/log.h>
+#include <zephyr/net/tls_credentials.h>
+#include <modem/location.h>
+#include <modem/modem_key_mgmt.h>
+
+LOG_MODULE_DECLARE(location, CONFIG_LOCATION_LOG_LEVEL);
+
+#if (!defined(CONFIG_LOCATION_METHOD_CELLULAR_EXTERNAL) &&\
+	!defined(CONFIG_LOCATION_METHOD_WIFI_EXTERNAL))
+
+/* TLS certificate:
+ *	CN=Starfield Services Root Certificate Authority - G2
+ *	O=Starfield Technologies, Inc.
+ *	OU=Starfield Class 2 Certification Authority
+ *	Serial number=a7:0e:4a:4c:34:82:b7:7f
+ *	Valid from=Sep 2 00:00:00 2009 GMT
+ *	Valid to=Jun 28 17:39:16 2034 GMT
+ *	Download url=https://www.amazontrust.com/repository/SFSRootCAG2.pem
+ */
+static const char tls_certificate_nrf_cloud[] =
```

Comment:
should this cert live in the nrf_cloud lib?
also what is the use case for this?  
to do MQTT or REST with nrf_cloud, the device must already have a device certificate and/or a private key in the NRF_CLOUD_SEC_TAG location... is it just in case a user forgets to include the nrf cloud CA?

Reply:
I guess the purpose of the CA certificate has been REST API usage. With the REST API, the CA certificate is the only certificate which is needed (in addition to the key needed to sign JWTs). But still, I'm not sure whether we need to have it in the library.

Reply:
I've removed certificate handling now.
I think this was original more for convenience in quick testing and I guess our tools and instructions w.r.t. certificate uploading etc. have also improved a lot.

Reply:
HERE certificate restored for convenience.